PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,3 +325,13 @@ def miniAOD_customizeAllMC(process):
     miniAOD_customizeCommon(process)
     miniAOD_customizeMC(process)
     return process
+
+def miniAOD_forPhase2(process):
+    print "Switching off electron and photon IDs (not yet working for Phase2 era)"
+    process.patElectrons.addElectronID = False
+    process.patElectrons.electronIDSources = cms.PSet()
+    process.patPhotons.addPhotonID = False 
+    process.patPhotons.photonIDSources = cms.PSet()
```

Comment:
What I'm trying to explain is that this change would make things nicer only today when there's no valid e/g id or modification for phase2 neither in miniAOD nor in PAT.
When at any moment there will be a e/g id or modification for phase2, then we'll still face the same problem, as just disabling the features in phase2 will no longer work.
So, i'd rather get a real solution to the issue, either by having customizes become better aware of the era or by avoiding the customizes, and in the meantime keep a `miniAOD_forPhase2` function which is a clear placeholder for a better solution (instead of scattering the temporary changes in 4 different files)